client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,226 @@
+.plan-thank-you-card {
+	position: relative;
+	width: 100%;
+	max-width: 500px;
+	background-color: darken( $alert-green, 10 );
+	border-radius: 8px 8px 6px 6px;
+	overflow: hidden;
+	margin: 56px auto 24px;
+	color: $white;
+	font-family: $sans;
+	text-align: center;
+
+	@media ( max-width: 500px ) {
+		margin-top: 0;
+	}
+}
+
+.plan-thank-you-card__header {
+	background-color: $alert-green;
+	padding: 75px 0 30px;
+	position: relative;
+	z-index: 1;
+	line-height: 1.2;
+	border-radius: 6px 6px 0 0;
+}
+
+.plan-thank-you-card__body {
+	padding: 40px;
+}
+
+.plan-thank-you-card__heading {
+	font-size: 22px;
+	font-weight: 600;
+	margin-bottom: 8px;
+}
+
+.plan-thank-you-card__description {
+	font-size: 15px;
+	font-weight: 400;
+	width: 280px;
+	max-width: 100%;
+	margin: 0 auto 30px;
+	line-height: 20px;
+}
+
+.plan-thank-you-card__button {
+	display: inline-block;
+	border-radius: 4px;
+	background-color: $white;
+	color: darken( $alert-green, 30 );
+	font-size: 14px;
+	height: 40px;
+	line-height: 40px;
+	padding: 0 60px;
+	border-style: solid;
+	border-color: darken( $alert-green, 20 );
+	border-width: 1px 1px 2px;
+
+	//box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.3),
+	//			0 2px 0 0 rgba(0, 0, 0, 0.3);
+
+	@media ( max-width: 320px) {
+		padding: 0;
+		width: 100%;
+		text-align: center;
+	}
+}
+
+.plan-thank-you-card__button:hover {
+	//box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.4),
+	//			0 2px 0 0 rgba(0, 0, 0, 0.4);
+	border-color: darken( $alert-green, 30 );
+	color: darken( $alert-green, 10 );
+}
+
+.plan-thank-you-card__button:focus {
+	color: darken( $alert-green, 10 );
+	outline: 1px dotted $white;
+}
+
+.plan-thank-you-card__button:active {
+	//box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.2),
+	//			0 -2px 0 0 rgba(0, 0, 0, 0.2);
+	border-width: 2px 1px 1px;
+	color: darken( $alert-green, 30 );
+}
+
+.plan-thank-you-card__plan-name {
+	font-size: 18px;
+	font-weight: 500;
+	margin-bottom: 5px;
+
+	&.is-placeholder {
+		@include placeholder();
+
+		background-color: #DAF0E0;
+		margin-left: auto;
+		margin-right: auto;
+		width: 35%;
+	}
+}
+
+.plan-thank-you-card__plan-price {
+	font-size: 15px;
+	font-weight: 500;
+	color: #DAF0E0;
+
+	&.is-placeholder {
+		@include placeholder();
+
+		background-color: #DAF0E0;
+		margin-left: auto;
+		margin-right: auto;
+		width: 10%;
+	}
+}
+
+@keyframes floating {
+	0% {
+		opacity: 0;
+		transform: translateY( 0 ) translateX( 0 ) rotate( 0 );
+	}
+	70% {
+		opacity: 1;
+		transform: translateY( 5px ) translateX( 5px ) rotate( -20deg );
+	}
+	100% {
+		opacity: 0;
+		transform: translateY( 0 ) translateX( 0 ) rotate( 0 );
+	}
+}
+
+.plan-thank-you-card__background-icons {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: -1;
```

Comment:
Let's see if we can avoid adding a stacking context here. Otherwise, lets add add this to our z-index map.

see: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#z-index

and https://github.com/gwwar/z-context for help in getting the right parent stacking context